.github/ dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
```

Comment:
It seems that the filepath has a whitespace (.github/%20dependabot.yml). Can you correct it before merging this?

Reply:
Yess, good catch 💯